test/integration_tests/long/test_similarity.py

```diff
@@ -388,3 +392,39 @@ def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_qdrant(sel
 
         res_batch = execute_query_fetch_all(self.evadb, select_query)
         self.assertEqual(res_batch.frames["testsimilarityimagedataset._row_id"][0], 5)
+
+        # Cleanup
+        self.evadb.catalog().drop_index_catalog_entry("testQdrantIndexImageDataset")
+
+    def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_pinecone(
+        self,
+    ):
+        # Set the env variables.
+        original_pinecone_key = os.environ.get("PINECONE_API_KEY")
+        original_pinecone_env = os.environ.get("PINECONE_ENV")
+
+        os.environ["PINECONE_API_KEY"] = "657e4fae-7208-4555-b0f2-9847dfa5b818"
+        os.environ["PINECONE_ENV"] = "gcp-starter"
```

Comment:
Crated this free tier API key just for EvaDB testing sake.

Reply:
You can use `self.evadb.config.update_value("third_party", "PINECONE_API_KEY", "657e4fae-7208-4555-b0f2-9847dfa5b818")` to update the value. You may also want to move this logic into cls setup and teardown. Otherwise, if the test failed, the environment variable will not be recovered.

Reply:
Used environment vars itself in setup and teardown